io/disk/ioping.py.data/README

```diff
@@ -0,0 +1,37 @@
+A tool to monitor I/O latency in real time. It shows disk latency in the same way as ping shows network latency.
+Homepage: https://github.com/koct9i/ioping/ (migrated from http://code.google.com/p/ioping/)
+Please send your patches, issues and questions to https://github.com/koct9i/ioping/issues/
+
+Inputs Needed in yaml file:
+---------------------------
+count - stop ioping after <count> requests
+deadline - stop ioping after <deadline> requests
+period - print raw statistics for every <period> requests
+interval - interval between requests in seconds
+size - request size
+wsize - working set size
+disk - path for the disk
+
```

Comment:
@PavamanSubramaniyam can u add little elaborated info for parameter like
`size` size of what? and what kind of request.
it will also more helpfull if you can add some generic example values.